README.md

```diff
@@ -17,3 +17,48 @@ Since this is a GitHub pages site, built CSS bundle must be committed to the rep
 1. Install Jekyll `gem install jekyll`
 2. Run `jekyll serve --watch --baseurl=''` and view on http://localhost:4000/
 3. Run `grunt watch` in parallel if you're editing styles
+
+## Documentation style guide
+This guide is based on https://jacobian.org/writing/great-documentation/. It exists as guidance to help keep our documentation consistent.
+
+### Types of documentation
+There are three types of documentation:
+1. _Step by step tutorials_: These are the on-ramp for our project. Users should be able to follow one of our step by steps and achieve success within 30 minutes. These live in this project.
```

Comment:
For some reason when viewing these bullet points, they didn't format properly. Might need a line break above?